core/src/main/scala/org/bitcoins/core/api/keymanager/KeyManagerCreateApi.scala

```diff
@@ -29,33 +30,38 @@ trait BIP39KeyManagerCreateApi[T <: BIP39KeyManagerApi]
     * $initialize
     */
   final def initialize(
+      aesPassword: AesPassword,
```

Comment:
I think we should allow this to be an `aesPasswordOpt: Option[AesPassword]`  ?

The reason I say this is if you have a bip39Password, your seed stored unencrypted on disk is ok as you can't do anything with just the unencryptd seed.